docs/mayflower.md

```diff
@@ -1,50 +1,67 @@
 3. Build the artifacts from Mayflower Patternlab by running `rush build:patternlab`.
 4. Check to see if the mayflower artifacts generated from the previous step exists in the your local openmass repo at openmass/docroot/libraries/mayflower-dev
 5. Run `ahoy drush cr` on the Drupal site to have the development artifacts picked up.
 
-Note that you will want to remove the `mayflower-dev` folder when you are finished development, since it will override whatever version of Mayflower is otherwise specified.
+> Note that you might want to remove the `mayflower-dev` folder when you are finished development, since it will override whatever version of Mayflower is otherwise specified.
+
+
+#### Using a branch of mayflower-artifacts in your Drupal branch:
+If you're working on a ticket that requires Mayflower changes and the Mayflower PR hasn't been merged into develop, you can _temporarily_ pull in a remote feature branch of mayflower-artifacts to Drupal for testing.
+
+1. From your terminal, within Docker, update and download your new mayflower version by running 
+```composer require massgov/mayflower-artifacts:dev-<your-mayflower-branch-name> --update-with-dependencies```
+> For example, if your Mayflower branch name is `DP-8411-test-branch`, after the branch has been deployed by circleCI, you can run `composer require massgov/mayflower-artifacts:dev-DP-8411-test-branch --update-with-dependencies` in openmass to pull it in.
+1. Commit the composer.json and composer.lock changes which correspond to updating mayflower-artifacts
+1. You should now have Mayflower updated in your feature branch. Remember to rebuild your cache!
+1. To deploy the Mayflower changes in openmass to a feature environment for external review, you just push your composer commit to Github
+    1. Tugboat will automatically build your branch with the mayflower artifacts version that you specified (Note: you will need to resolve any conflicts on the branch with develop or the deployment will fail)
+    2. To deploy to an Acquia feature environment, run `drush ma:deploy <feature-environment> <your-openmass-branch-name>` inside the SSH shell. 
+
+If you have committed your change to point to a feature branch of mayflower artifacts instead of the develop branch, make sure you change it back to point to `dev-develop` before merging into develop. 
+
+#### Merging a feature PR with Mayflower changes:
+All openmass feature development PRs that require Mayflower changes must pass all these checks before squashing and merging into develop:
+- [ ] PR reviewed and approved 
+- [ ] Corresponding Mayflower PR is merged into develop
+- [ ] Mayflower artifacts is pointing to the `dev-develop` version in composer.json
+- [ ] Up-to-date with latest openmass develop branch (resolve any composer.lock conflicts with the develop branch)
+- [ ] All circleCI tests are green on your branch
```

Comment:
This content might be more appropriate for the peer review checklist, which is linked at bottom of every PR. But we can do that in a followup.

Reply:
make sense, I moved the checklist into the peer review doc 4160130